platform-sdk/docs/proposals/template-2.md

```diff
@@ -0,0 +1,184 @@
+# Proposal Title Goes Here
+
+| Metadata           | Entities                            | 
+|--------------------|-------------------------------------|
+| Designers          | John, Mary, alpha-team              |
+| Functional Impacts | Services, DevOps, Mirror Node, etc. |
+| Related Proposals  | Proposal-1, Proposal-2              |
+| HIPS               | HIP-1, HIP-2,                       |
+| State              | DRAFT                               |
```

Comment:
Same and comment below - this is tracked in the board and should not be duplicated here.

Reply:
ok, will remove